static/css/_global.scss

```diff
@@ -297,13 +326,37 @@ p.error {
 body > footer {
   display: flex;
   align-content: center;
```

Comment:
Under that we could also add the line:
`display: -webkit-flex;` to ensure Safari likes it :-)

Reply:
According to [this table](https://developer.mozilla.org/en-US/docs/Web/CSS/display#Browser_compatibility) Safari supports `display: flex` even without a specific prefix.